docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift

```diff
@@ -0,0 +1,148 @@
+        let dict: [String: Any] = ["type": "task",
+                                   "owner": "todo",
+                                   "createdAt": Date()]
+        let newTask = Document(dictionary: dict)
+        try? database.save(newTask)
+        
+        // mutate document
+        newTask.set("Apples", forKey:"name")
+        try? database.save(newTask)
+        
+        // typed accessors
+        newTask.set(Date(), forKey: "createdAt")
+        let date = newTask.date(forKey: "createdAt")
+        
+        // database transaction
+        do {
+            try database.inBatch {
+                for i in 0...10 {
+                    let doc = Document()
+                    doc.set("user", forKey: "type")
+                    doc.set("user \(i)", forKey: "name")
+                    try database.save(doc)
+                    print("saved user document \(doc.string(forKey: "name"))")
+                }
+            }
+        } catch let error {
+            print(error.localizedDescription)
+        }
+        
+        // blob
+        let appleImage = UIImage(named: "avatar.jpg")!
+        let imageData = UIImageJPEGRepresentation(appleImage, 1)!
+        
+        let blob = Blob(contentType: "image/jpg", data: imageData)
+        newTask.set(blob, forKey: "avatar")
+        try? database.save(newTask)
+        
+        if let taskBlob = newTask.blob(forKey: "image") {
+            UIImage(data: taskBlob.content!)
+        }
+        
+        // query
+        let query = Query
+            .select()
+            .from(DataSource.database(database))
+            .where(
+                Expression.property("type").equalTo("user")
+                    .and(Expression.property("admin").equalTo(false))
+        )
+        
+        do {
+            let rows = try query.run()
+            for row in rows {
+                print("doc ID :: \(row.documentID)")
+            }
+        } catch let error {
+            print(error.localizedDescription)
+        }
+        
+        // fts example
+        // Insert documents
+        let tasks = ["buy groceries", "play chess", "book travels", "buy museum tickets"]
+        for task in tasks {
+            let doc = Document()
+            doc.set("task", forKey: "type")
+            doc.set(task, forKey: "name")
+            try? database.save(doc)
+        }
+        
+        // Create index
+        do {
+            try database.createIndex(["name"], options: .fullTextIndex(language: nil, ignoreDiacritics: false))
+        } catch let error {
+            print(error.localizedDescription)
+        }
+        
+        let whereClause = Expression.property("name").match("'buy'")
+        let ftsQuery = Query.select().from(DataSource.database(database)).where(whereClause)
+        
+        do {
+            let ftsQueryResult = try ftsQuery.run()
+            for row in ftsQueryResult {
+                print("document properties \(row.document.toDictionary())")
```

Comment:
We don't have .document property in DB13. The document ID projection is needed.

Note: We are still discussing about how to get the document from the query result.